docs/_docs/security/sandbox.adoc

```diff
@@ -0,0 +1,80 @@
+= The Ignite Sandbox
+
+== Overview
+Ignite allows using a user-defined code that can be a compute job, event filter, message listener, etc.
```

Comment:
How about this version for the opening paragraph?

Ignite allows executing custom logic via various APIs including compute tasks, event filters, message listeners. This user-defined logic can utilize Java APIs to get access to host resources. For example, it can create/update/delete files or system properties, open network connections, use reflection and other APIs to get full control of the host environment. Ignite Sandbox is based on the [Java Sandbox model](https://docs.oracle.com/en/java/javase/11/security/java-se-platform-security-architecture.html#GUID-C203D80F-C730-45C3-AB95-D4E61FD6D89C) and allows you to restrict the scope of user-defined logic executed via Ignite APIs.

Reply:
Great I like your version!